dags/vm_resource.py

```diff
@@ -165,6 +165,9 @@ class DockerImage(enum.Enum):
       "gcr.io/tpu-prod-env-multipod/maxtext_jax_stable:"
       f"{datetime.datetime.today().strftime('%Y-%m-%d')}"
   )
+  MAXTEXT_TPU_JAX_SS = (
```

Comment:
Would like to double check if we expect one pinned version for now. Thanks!

Reply:
Yes, that is correct.